packages/http-client/tests/client.spec.ts

```diff
@@ -475,14 +483,30 @@ describe('client', () => {
       }
     })
 
-    it('returns empty exchanges array if response is ok and body is empty array', async () => {
+
+    it('returns array of message exchanges given a list of exchange IDs', async () => {
+      const alice = await DidJwk.create()
+      const aliceRfq = await DevTools.createRfq({ sender: alice })
+      await aliceRfq.sign(alice)
+
+      const bob = await DidJwk.create()
+      const bobRfq = await DevTools.createRfq({ sender: bob })
+      await bobRfq.sign(bob)
```

Comment:
maybe what would make more sense naming wise is to say `aliceRfq1` and `aliceRfq2`, since the GET /exchanges call would be returning all exchanges for a given did? so maybe something like 

```suggestion
      const aliceRfq2 = await DevTools.createRfq({ sender: alice })
      await aliceRfq2.sign(alice)
```

Reply:
Done. Thanks @jiyoontbd for the knowledge, makes sense, didn't realize the `did` property was meant to be for the caller, I've updated that too, let me know if I misunderstood again!